spec/rubocop/cop/ezcater/feature_flag_name_valid_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Ezcater::FeatureFlagNameValid, :config do
+  subject(:cop) { described_class.new(config) }
+
+  before { inspect_source(line) }
+  describe "assignment to a constant" do
+    context "with a valid feature flag name" do
+      let(:line) { %[MY_FLAG = "Foo::Bar"] }
+
+      it "does not report an offense" do
+        expect(cop.offenses).to be_empty
+      end
+    end
+
+    context "with an invalid constant name" do
```

Comment:
🎱 To match the other contexts.

```suggestion
    context "with an invalid feature flag name" do
```

Reply:
Thank you!